Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml

```diff
@@ -188,26 +188,27 @@ tasks:
     isautoswitchedtoquietmode: false
   "11":
     id: "11"
-    taskid: 97a8b99b-265b-493a-8a3c-5f22b17f5fc2
-    type: title
+    taskid: 322faa9a-3a45-476d-88ac-ba4cf2bcc9d5
```

Comment:
Please edit playbook so:
- email should only be sent for full AR
- Also, you need to add a conditional to check to make sure the service owner has an email.